src/L2DebtManager.sol

```diff
@@ -261,8 +266,33 @@ contract L2DebtManager is
     /**
      * @inheritdoc IL2DebtManager
      */
-    function totalBorrowingAmount() external view returns (uint256) {
-        return _totalBorrowingAmount;
+    function totalBorrowingAmount() public view returns (uint256) {
+        uint256 accInterestAlreadyStored = _accumulatedInterestPerDebtTokenStored;
+
+        return
+            ((
+                (1e18 *
+                    _totalBorrowingAmount *
+                    (accumulatedInterestPerDebtToken() -
+                        accInterestAlreadyStored))
+            ) /
+                1e20 +
+                1e18 *
+                _totalBorrowingAmount) / 1e18;
```

Comment:
This calculation is being repeated. can we wrap it to an internal function?

Reply:
Done